manim/mobject/svg/opengl_text_mobject.py

```diff
@@ -725,7 +725,7 @@ class OpenGLMarkupText(OpenGLSVGMobject):
 
     You can find more information about Pango markup formatting at the
     corresponding documentation page:
-    `Pango Markup <https://developer.gnome.org/pango/stable/pango-Markup.html>`_.
+    `Pango Markup <https://developer.gnome.org/pygtk/stable/pango-markup-language.html>`_.
```

Comment:
irrc, the above one is the correct link, it is just that people aren't willing to redirect. instead, you should change it to https://developer.gnome.org/pango/1.46/pango-Markup.html and not the one with pygtk.

Reply:
Also, in `manim/mobject/svg/text_mobject.py`

Reply:
Thank you! I had a feeling I didn't get quite the right one but didn't want to hard code the version. Addressed in #1542 since this is merged already.

Reply:
See https://gitlab.gnome.org/Infrastructure/library-web/-/issues/98